benchkit/commandwrappers/trace-cmd.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023 Huawei Technologies Co., Ltd. All rights reserved.
```

Comment:
avoid the dash `-` in module names, it makes the module harder to import. A simple `tracecmd.py` would do.